log/keyvalue_test.go

```diff
@@ -365,40 +365,40 @@ func TestAllocationLimits(t *testing.T) {
 		m     []log.KeyValue
 	)
 
-	assert.Equal(t, 0.0, testing.AllocsPerRun(runs, func() {
+	assert.InDelta(t, 0.0, testing.AllocsPerRun(runs, func() {
```

Comment:
`testing.AllocsPerRun` always returns an integer. It is documented and the Go standard library in its unit tests also always compares its result with a constant value. See https://github.com/search?q=repo%3Agolang%2Fgo%20AllocsPerRun&type=code